docs/source/extensions.rst

```diff
@@ -0,0 +1,61 @@
+.. _extensions-catalog:
+
+********************************
+Extensions Catalog and Proposals
+********************************
+
+NWB Extensions (NDX) Catalog
+============================
+
+The :ndx-catalog:`NDX Catalog website <>` contains published
+extensions created by the community to add support for a wide variety of data types.
+Before creating your own extension, we recommend that you explore the Catalog to
+make sure that someone has not already created an extension that suits your needs
+or that you could build off of.
+
+Policies on Community-driven Development and Integration of New Neuroscience Technologies with NWB
```

Comment:
I would make this a subsubheading. If not a subsubheading then I would change the punctuation to regular lower-case

```suggestion
Policies on Community-driven Development and Integration of New Neuroscience Technologies with NWB
```